stylix/palette.nix

```
@@ -5,27 +5,57 @@
   options,
   ...
 }:
-
 let
   cfg = config.stylix;
   opts = options.stylix;
 in
 {
+  imports = [
+    (lib.mkRenamedOptionModule
+      [ "stylix" "polarity" ]
+      [ "stylix" "theme" "polarity" ]
+    )
+  ];
```

If https://github.com/nix-community/stylix/pull/1706 goes through, this should be reformatted as:

```suggestion
  imports = lib.singleton (
    lib.mkRenamedOptionModule
      [ "stylix" "polarity" ]
      [ "stylix" "theme" "polarity" ]
    );
```